### ISSUE TITLE
Unable to install R packages

### ISSUE THREAD

**Issue by SandeepNadella** [campsych/concerto-platform#250]
### Concerto Platform version
v5.0.beta.10.1

### Actual behavior
Tried installing RCurl package under R packages section of Administration tab but the installation is stuck with the following output
```
Output:

checking...
checks passed
task #1 started
```
I used 
**Name**: RCurl
**Mirror URL**: https://cran.r-project.org/package=RCurl while installing.

1. What should be the mirror URL while installing R packages?
1. How to stop/cancel the installation since I get the following error message while trying to install another package?
```
R package installation failed
There are already pending or ongoing scheduled tasks.
```
The installation status has been **Ongoing** since long(like an hour).

### Expected behavior
The R package must be installed and ready for use in the code type test 
```
library(RCurl)
result <- postForm(
    uri='https://some_api_url/',
    token='Some_Token',
    content='some_content',
    format='json',
    returnFormat='json'
)
print(result)
```

**Comment by anusornk** [comment]
I’m also dealing with the same problem.

**Comment by anusornk** [comment]
I solved the problem by adding these commands to the Dockerfile 

RUN R -e "install.packages ('RCurl', dependencies = TRUE, repos = 'http://cran.us.r-project.org')"

Then compose up again.
docker-compose up --build

**Comment by SandeepNadella** [comment]
Thanks @anusornk. 
I tried installing the package from docker terminal and it worked.
I tried from the UI and installation of the first package was successful but the status wouldn't change to Complete from Ongoing. Since this doesn't change to complete I couldn't install new packages.

**Comment by Foshb** [comment]
I am having the same issue on an Amazon EC2 server. Is there a solution for this too?

**Comment by przemyslawlis** [comment]
Fixed in 876cfdb35c709eaca6cf61aa940ef06f25f4937c.
Fix will be part of 5.0.12 release.
